nomad/api/variables.py

```diff
@@ -0,0 +1,66 @@
+import nomad.api.exceptions
+
+from nomad.api.base import Requester
+
+
+class Variables(Requester):
```

Comment:
Thoughts on implementing `__iter__`?

Reply:
Yes, it's a good point, thanks. I implemented `__iter__`